security/seccomp-profiles.adoc

```diff
@@ -7,13 +7,11 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 An {product-title} container or a pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
-Seccomp, secure computing mode, is a  Linux kernel feature that can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating a profile that is applied to a container or pod.
-Seccomp profiles are stored as JSON files on the disk.
+Seccomp, secure computing mode, is a  Linux kernel feature that can be used to limit the process running in a container to only call a subset of the available system calls. 
```

Comment:
- I think this sentence would read better as "Secure computing mode, secomp,..." but it's minor.
- To avoid having "call(s)" used twice, could we end with: "... to only using a subset of the available system calls."